### PR TITLE
updating screenshots for rebranding

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/import-postman-collection.md [hunk 1]
```diff
@@ -23,8 +23,9 @@ Looking to import from a spec file? See [Building a Test from a Spec File](/api-
 
 1. Log in to Sauce Labs, then click **API Testing**.
 2. Click to open one of your Projects.
-3. Click the **HTTP Client** tab.<br/><img src={useBaseUrl('img/api-fortress/2021/01/HTTPClient.png')} alt="HTTP Client" width="400" />
-4. Click the **Import OpenAPI/HAR/Postman** button, then select and upload your Postman Collection file from your local machine.<br/><img src={useBaseUrl('img/api-fortress/2021/01/importSpec.png')} alt="Import OpenAPI / Postman button" width="250"/>
+3. Click the **HTTP Client** tab.<br/><img src={useBaseUrl('img/api-testing/HTTPClient_new_brand.png')} alt="HTTP Client" width="400" />
+4. Click the **Import OpenAPI/Postman** button, then **Import OpenAPI/Postman Collection/.har** and then, select and upload your Postman Collection file from your local machine.<br/><img src={useBaseUrl('img/api-testing/importSpec.png')} alt="Import OpenAPI / Postman button" width="250"/>
```
Comment (aliciakrebs) [inline docs/api-testing/import-postman-collection.md:27]: this image is missing

File: docs/api-testing/import-postman-collection.md [hunk 2]
```diff
@@ -34,85 +35,108 @@ If you don't have a file available, try out the sample below.
...
 }
 ```
 
 </details>
 
 :::
 
-5. Click the folder your **Snapshots** tree where you'd like to save your file.<br/><img src={useBaseUrl('img/api-fortress/2021/04/importFolder.png')} alt="Routes Rendered"/>
-6. Click **Save**.<br/><img src={useBaseUrl('img/api-fortress/2021/04/importFolder2.png')} alt="Import file to Project"/>
+5. Click the folder your **Snapshots** tree where you'd like to save your file.<br/><img src={useBaseUrl('img/api-testing/importFolder.png')} alt="Routes Rendered"/>
```
Comment (aliciakrebs) [inline docs/api-testing/import-postman-collection.md:103]: image is missing

File: docs/api-testing/import-postman-collection.md [hunk 3]
```diff
@@ -34,85 +35,108 @@ If you don't have a file available, try out the sample below.
...
+6. Click **Save**.<br/><img src={useBaseUrl('img/api-testing/importFolder2.png')} alt="Import file to Project"/>
```
Comment (aliciakrebs) [inline docs/api-testing/import-postman-collection.md:104]: image is missing

**Comment by LoryApiFortress** [comment]
@aliciakrebs The branch has conflicts that require time to resolve so I'm closing this PR and creating a new one

**Pull request closed without merging**